storage-node/src/commands/server.ts

```diff
@@ -91,8 +91,8 @@ export default class Server extends ApiCommandBase {
     queryNodeEndpoint: flags.string({
       char: 'q',
       required: true,
-      default: 'http://localhost:8081/graphql',
-      description: 'Query node endpoint (e.g.: http://some.com:8081/graphql)',
+      default: 'http://localhost:4352/graphql',
```

Comment:
If the flag is required does default value make any difference?

Reply:
> Actually if the flag is required does default value make any difference?

If you don't provide the value, then default value will be used, otherwise (if default value was not set) the command will throw an error that required flag is missing